Various technical notes/transition_infos.json

```diff
@@ -690,11 +678,6 @@
           "area_name": "Crystal Cavern",
           "requires": []
         },
```

Comment:
Why remove this?

```suggestion
        },
        {
          "area_id": "0x4B08BBEB",
          "area_name": "Scorpion Temple (Harry)",
          "requires": []
        },
```

Reply:
As mentioned in both the commit titles as well as the utils.py file that writes it to the Spoiler Log:
**Scorpion Temple (Harry)** is temporarily removed from the randomization proccess. An essential part of that removal is that this exit is not included in the .json file.

Reply:
All temporarily removed exits are put back in the json file, but randomizer is not affected.